packages/editor-ui/src/components/CredentialCard.vue

```diff
@@ -213,5 +225,16 @@ function moveResource() {
 	align-self: stretch;
 	padding: 0 var(--spacing-s) 0 0;
 	cursor: default;
+
+	@include mixins.breakpoint('sm-and-down') {
+		width: 100%;
+		padding: 0;
+	}
+}
+
+.cardBadge {
+	@include mixins.breakpoint('sm-and-down') {
+		margin-right: auto !important;
```

Comment:
We could get rid of this `!important` if we'd use a regular css instead of `mr-xs` in ProjectCardBadge, or use css variable

Reply:
Done